app/js/lib/foxbox/sequential-timer.js

```diff
@@ -0,0 +1,94 @@
+const p = Object.freeze({
```

Comment:
Yes, what about ```shared``` like react does [1] or ```common``` like in angular [2]?

Honestly I much more like how angular is organized in general comparing to react, so ```common``` sounds better to me, but I'll choose whatever you prefer.

[1] https://github.com/facebook/react/tree/master/src/shared
[2] https://github.com/angular/angular/tree/master/modules/angular2/src/common